src/js/tosdr.js

```diff
@@ -90,10 +89,38 @@ $(function () {
         $('html, body').animate({
           scrollTop: $("#services").offset().top
         }, 1000);
-      }
+      } 
     } else {
       showModal(hash);
     }
   }
+  
+  if (location.search.indexOf('?') == 0) {
+    var GETarguments = location.search.substring(1)
+    var splits = GETarguments.split("=");
+    if (splits.length == 2 && splits[0] == "servicePoint" && splits[1].length > 0) {
```

Comment:
On MDN there's a more robust way to get query parameters: https://developer.mozilla.org/en-US/docs/Web/API/URLUtils/search#Get_the_value_of_a_single_search_param

Especially considering how we're going to be accepting reviews by visitors, I think it's good to make sure that this will keep working regardless of the ID of the service, so I think it would be nice if you used that to find the `servicePoint` parameter's value :)